plugins/modules/cloud/lxc/lxc_container.py

```diff
@@ -1442,11 +1427,10 @@ def _rsync_data(self, container_path, temp_dir):
                 self.module.get_bin_path('rsync', True),
                 '-aHAX',
                 fs_path,
-                temp_dir
+                temp_dir,
             ]
-            rc, stdout, err = self._run_command(
+            rc, stdout, err = self.module.run_command(
                 build_command,
-                unsafe_shell=True
```

Comment:
Why is it safe to remove it here?

Reply:
Because all four arguments are single elements, already as strings. No need for shell parsing on any of them.

Reply:
I'm wondering more why it was in there in the first place. It doesn't really make sense to me...

Reply:
someone going overkill, maybe?